src/components/atoms/Image.tsx

```diff
@@ -0,0 +1,65 @@
+import { cva } from "cva";
+import { cn } from "../../utils/cn";
+
+const imageVariants = cva(
+  "h-full w-full rounded-md object-cover object-center",
+  {
+    variants: {
+      aspect: {
+        Square: "aspect-square",
+        Rectangle: "aspect-video",
+        None: "",
+      },
+      size: {
+        Small: "w-40",
+        Medium: "w-96",
+        Large: "w-full",
+      },
+    },
+  }
+);
+
+export interface ImageProps {
+  /**
+   * @displayName Source
+   * @tooltip The URL of the image to display
+   */
+  src: string;
+  /**
+   * @displayName Alternate Text
+   * @tooltip Controls the alternate text of the img tag
+   */
+  alt: string;
+  /**
+   * @displayName Aspect Ratio
+   * @tooltip Controls the aspect ratio of the image
+   */
+  aspect: "Square" | "Rectangle" | "None";
+  /**
+   * @displayName Size
+   * @tooltip Controls the size of the image
+   */
+  size?: "Small" | "Medium" | "Large";
+}
+
+export const initialProps: ImageProps = {
+  src: "https://a.mktgcdn.com/p/2L6Bp69LgXuwu_TPYmkvWNy24VxmOpmae33-f4fLlM8/1920x1280.jpg",
```

Comment:
Can you have this point to https://placehold.co/1920x1280? Just so this doesn't break if the image is deleted or changed in the yext account

Reply:
done